tests/cpp_unit_tests/test_tap_position_optimizer.cpp

```diff
@@ -304,6 +304,28 @@ TEST_CASE("Test Transformer ranking") {
             }
         }
 
+        SUBCASE("Multiple source grid") {
+            // Grid with multiple sources and symetric graph
+            pgm_tap::TrafoGraphEdges edge_array = {{0, 1}, {1, 2}, {2, 3}, {3, 4}, {4, 5}};
+            pgm_tap::TrafoGraphEdgeProperties edge_prop{
+                {{0, 1}, 1}, {{1, 2}, 1}, {{2, 3}, 1}, {{3, 4}, 1}, {{4, 5}, 1}};
+            std::vector<pgm_tap::TrafoGraphVertex> vertex_props{{true}, {false}, {false}, {false}, {false}, {true}};
+
+            pgm_tap::TransformerGraph g{boost::edges_are_unsorted_multi_pass, edge_array.cbegin(), edge_array.cend(),
+                                        edge_prop.cbegin(), 6};
+
+            // Vertex properties can not be set during graph creation
+            boost::graph_traits<pgm_tap::TransformerGraph>::vertex_iterator vi, vi_end;
+            for (boost::tie(vi, vi_end) = vertices(g); vi != vi_end; ++vi) {
+                g[*vi].is_source = vertex_props[*vi].is_source;
+            }
+
+            pgm_tap::TrafoGraphEdgeProperties const regulated_edge_weights = get_edge_weights(g);
+            pgm_tap::TrafoGraphEdgeProperties const ref_regulated_edge_weights{
+                {{0, 1}, 0}, {{1, 2}, 1}, {{2, 3}, 2}, {{3, 4}, 2}, {{4, 5}, 1}};
```

Comment:
shouldn't this be
```suggestion
                {{0, 1}, 0}, {{1, 2}, 1}, {{2, 3}, 2}, {{3, 4}, 1}, {{4, 5}, 0}};
```
?

Reply:
Edge source/target ambiguity fixed.